src/middleware.js

```diff
@@ -65,3 +65,18 @@ function getExpressRequestMethod(route) {
   }
   return null;
 }
+
+function getExpressRequestPath(route) {
+  if (route) {
+    switch (typeof(route.path)) {
+      case 'string':
+        return route.path;
+      case 'object':
+        if (route.path.length > 0) {
```

Comment:
If `route.path.length == 1` then what?

Reply:
There just isn't a path?

Reply:
Ideally, that should not happen in my opinion. I had a choice to send an empty string as `route` dimension but I chose to use null instead.

Reply:
It it's an object (an array, actually), the path exists in multiple pieces and they need to be concatenated.